src/plugins/webPage/components/webPage.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+    <div class="l-iframe abs">
+        <iframe :src="currentDomainObject.url"></iframe>
+    </div>
+</template>
+
+<script>
+export default {
+    inject: ['openmct', 'domainObject'],
+    data: function () {
+        return {
+            currentDomainObject: this.domainObject
+        }
+    },
+    methods: {
+        updateDomainObject(newDomainObject) {
+            this.currentDomainObject = newDomainObject;
+        }
+    },
+    mounted() {
+        this.unobserve = this.openmct.objects.observe(this.domainObject, '*', this.updateDomainObject);
```

Comment:
You can remove the * observer here. I'm going to merge a PR soon that will remove them from everywhere else in the Open MCT codebase.

Reply:
The naming convention for JS files is UpperCamelCase until we decide otherwise. There has been some inconsistencies in the TCR branch so let's not add to add.  The file name should be WebPage.vue